src/Functions/custom/Functions.format.ps1xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Configuration>
+    <ViewDefinitions>
+        <View>
+            <Name>FunctionApp</Name>
+            <ViewSelectedBy>
+                <TypeName>Microsoft.Azure.PowerShell.Cmdlets.Functions.Models.Api20180201.Site</TypeName>
+            </ViewSelectedBy>
+            <TableControl>
+                <TableHeaders>
+                    <TableColumnHeader><Label>Name</Label></TableColumnHeader>
+                    <TableColumnHeader><Label>Status</Label><Width>7</Width></TableColumnHeader>
+                    <TableColumnHeader><Label>OSType</Label><Width>8</Width></TableColumnHeader>                    
+                    <TableColumnHeader><Label>RuntimeName</Label></TableColumnHeader>
+                    <TableColumnHeader><Label>Host</Label></TableColumnHeader>
+                    <TableColumnHeader><Label>Location</Label></TableColumnHeader>                    
+                    <TableColumnHeader><Label>AppServicePlan</Label></TableColumnHeader>
+                    <TableColumnHeader><Label>ResourceGroupName</Label></TableColumnHeader>
+                    <TableColumnHeader><Label>SubscriptionId</Label></TableColumnHeader>
+                </TableHeaders>
+                <TableRowEntries>
+                    <TableRowEntry>
+                        <TableColumnItems>
+                            <TableColumnItem><PropertyName>Name</PropertyName></TableColumnItem>
+                            <TableColumnItem><PropertyName>Status</PropertyName></TableColumnItem>
+                            <TableColumnItem><PropertyName>OSType</PropertyName></TableColumnItem>
+                            <TableColumnItem><PropertyName>RuntimeName</PropertyName></TableColumnItem>
+                            <TableColumnItem><PropertyName>HostVersion</PropertyName></TableColumnItem>
+                            <TableColumnItem><PropertyName>Location</PropertyName></TableColumnItem>                            
+                            <TableColumnItem><PropertyName>AppServicePlan</PropertyName></TableColumnItem>
+                            <TableColumnItem><PropertyName>ResourceGroupName</PropertyName></TableColumnItem>
+                            <TableColumnItem><PropertyName>SubscriptionId</PropertyName></TableColumnItem>
+                        </TableColumnItems>
+                    </TableRowEntry>
+                </TableRowEntries>
+            </TableControl>
+        </View>
+        <View>
+            <Name>FunctionAppPlan</Name>
+            <ViewSelectedBy>
+                <TypeName>Microsoft.Azure.PowerShell.Cmdlets.Functions.Models.Api20180201.AppServicePlan</TypeName>
+            </ViewSelectedBy>
+            <TableControl>
+                <TableHeaders>
+                    <TableColumnHeader><Label>Name</Label></TableColumnHeader>
+                    <TableColumnHeader><Label>WorkerType</Label></TableColumnHeader>                    
```

Comment:
Note that this is OK, but you can do the same changes using configuration only, see:https://github.com/Azure/autorest/blob/master/docs/powershell/directives.md#Table-Formatting

You don't need to change this tight now, but it is probably easier in the long run

Reply:
I've tried to make it work that way, but I ran into some issues. I can try to make it work on my next iteration.